src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -49,6 +49,11 @@ export function updateLayeredMaterialNodeImagery(context, layer, node, parent) {
     }
     const extentsDestination = node.getExtentsByProjection(layer.projection);
 
+    const zoom = extentsDestination[0].zoom;
+    if (zoom > layer.zoom.max || zoom < layer.zoom.min) {
+        return;
```

Comment:
Is it relevant to add a `noMoreUpdatePossible()` call ?

Reply:
in this step, `layerUpdateState` isn't instanced

Reply:
Yes, but what about moving it after the initialization ?

Reply:
because `MaterialLayer` must not be instanced because this node won't be never visible.